src/main/java/org/folio/list/services/ListService.java

```diff
@@ -128,8 +128,8 @@ public Optional<ListDTO> updateList(UUID id, ListUpdateRequestDTO request) {
     Optional<ListEntity> listEntity = listRepository.findById(id);
     listEntity.ifPresent(list -> {
       ListVersion previousVersions = new ListVersion();
-      previousVersions.setDataFromListEntity(listEntity.get(), getCurrentUser());
-      listVersionRepository.save(previousVersions);
+      previousVersions.setDataFromListEntity(listEntity.get());
```

Comment:
Nit: Since this is in the `listEntity.ifPresent()` lambda, the list is already available as `list`, so there's no need to use `listEntity.get()` to get it
```suggestion
      previousVersions.setDataFromListEntity(list);
```